graph2tac/loader/predict_server.py

```diff
@@ -138,6 +140,13 @@ class LoggingCounters:
     May incorrectly be 0.0 if there isn't enough time between 'initialize' calls.
     """
 
+    @contextmanager
+    def measure_build_network_time(self):
+        t0 = time.time()
+        yield
+        t1 = time.time()
+        self.build_network_time += t1 - t0
```

Comment:
Is this ever reset, or is this the cumulative time since the start of the server?

Reply:
Cumulative, I never reset this counter. The number of network recompilations should be little.